packages/objection/src/interpreters.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  CompoundCondition,
+  FieldCondition,
+  Comparable
+} from '@ucast/core';
+import { Query, ObjectionOperator } from './interpreter';
+
+export const $eq: ObjectionOperator<FieldCondition> = (condition, query) => {
+  return query.where(condition.field, '=', condition.value);
+};
+
+export const $ne: ObjectionOperator<FieldCondition> = (condition, query) => {
+  return query.where(condition.field, 'Not', condition.value);
+};
+
+export const $lt: ObjectionOperator<FieldCondition<Comparable>> = (condition, query) => {
+  return query.where(condition.field, '<', condition.value);
+};
+
+export const $lte: ObjectionOperator<FieldCondition<Comparable>> = (condition, query) => {
+  return query.where(condition.field, '<=', condition.value);
+};
+
+export const $gt: ObjectionOperator<FieldCondition<Comparable>> = (condition, query) => {
+  return query.where(condition.field, '>', condition.value);
+};
+
+export const $gte: ObjectionOperator<FieldCondition<Comparable>> = (condition, query) => {
+  return query.where(condition.field, '>=', condition.value);
+};
+
+export const $exists: ObjectionOperator<FieldCondition<Comparable>> = (condition, query) => {
+  return query.where(condition.field, 'NotNull', condition.value);
```

Comment:
Here we need to account case when we want to check that property does not exist

```suggestion
  return query.where(condition.field, condition.value ? 'NotNull' : 'Null');
```